src/color-swatch/color-swatch.js

```diff
@@ -109,6 +109,32 @@ const Self = class ColorSwatch extends NudeElement {
 			this.style.setProperty("--color", colorString);
 		}
 
+		if (name === "coords") {
+			if (!this.coords.length) {
+				return;
+			}
+
+			if (!this._el.coords) {
+				this._el.colorWrapper.insertAdjacentHTML("afterend", `<dl part="coords"></dl>`);
```

Comment:
Nit but I'd rather only insert this once and then keep it around in `this._el.coords`:

```js
this._el.coords ??= Object.assign(document.createElement("dl"), {part: "coords"});
```

Reply:
Q: Do we want to place `this._el.coords` inside `this._el.colorWrapper`? Previously, it was its sibling.

Reply:
No, use `.before()` instead